### PR TITLE
added Mean function to jax.numpy 

### PR DESCRIPTION
**Pull request by OkeyAmy**
# PR Description

## Related Issue

Close #

## Checklist

- [ ] Did you add a function?
- [ ] Did you add the tests?
- [ ] Did you run your tests and are your tests passing?
- [ ] Did pre-commit not fail on any check?
- [ ] Did you follow the steps we provided?

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by OkeyAmy** [comment]
Close #26054

**Comment by MuhammedAshraf2020** [comment]
Hello, @OkeyAmy 
I appreciate your contribution. Please inform me when you've completed your work so that I can review it with you.

**Comment by OkeyAmy** [comment]
Am done.

File: ivy/functional/frontends/jax/numpy/mathematical_functions.py [hunk 1]
```diff
@@ -726,3 +726,7 @@ def vdot(a, b):
 
 abs = absolute
 true_divide = divide
+
+@to_ivy_arrays_and_back
+def mean(x1, /):
```
Comment (MuhammedAshraf2020) [inline ivy/functional/frontends/jax/numpy/mathematical_functions.py:731]: This frontend function should have the same calling signature as the original function documented [here](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.mean.html)

**Review by MuhammedAshraf2020 — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/jax/numpy/mathematical_functions.py [hunk 2]
```diff
@@ -383,6 +383,11 @@ def maximum(x1, x2, /):
     return ivy.maximum(x1, x2)
 
 
+@to_ivy_arrays_and_back
+def mean(x, keepdims=False, out=None, axis=None, dtype=None, *, where=None):
+    return ivy.mean(x)
```
Comment (MuhammedAshraf2020) [inline ivy/functional/frontends/jax/numpy/mathematical_functions.py:388]: You have added the same call signature, which is great. However, you need to ensure that you handle and match all of these arguments and keywords with the ones in `ivy.mean`. For example, `ivy.mean` has arguments such as `axis`, `keepdims`, and `out`, which also exist in the frontend function. You should leverage these arguments in the ivy API `ivy.mean`.

For more information, you can refer to the frontend documentation here: [ivy Frontend Documentation](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends.html).

Additionally, here's an example of that function in another framework for reference: [Numpy Example](https://github.com/unifyai/ivy/blob/main/ivy/functional/frontends/numpy/statistics/averages_and_variances.py#L75).

**Review by MuhammedAshraf2020 — CHANGES_REQUESTED** [review]
